convert/convert.ml

```diff
@@ -53,11 +55,28 @@ let rec convert input_disks options source =
 
   message (f_"Opening the source");
   let g = open_guestfs ~identifier:"v2v" () in
-  g#set_memsize (g#get_memsize () * 2);
-  (* Setting the number of vCPUs allows parallel mkinitrd, but make
-   * sure this is not too large because each vCPU consumes guest RAM.
-   *)
-  g#set_smp (min 8 (Sysconf.nr_processors_online ()));
+  let memsize =
+    match options.memsize with
+    | None ->
+       (* Default (if [--memsize] option is not used) is to calculate
+        * some multiple of the libguestfs default memory size.
```

Comment:
How does libguestfs choose the default? ie in which cases would we need to use this? 
Maybe forklift could automatically override this default so we don't have the magical number `* 2`

Reply:
Currently it's just hard-coded, and we occasionally have increased it (for example: https://github.com/libguestfs/libguestfs/commit/224f373043302845122bf701ffc6e3416e0168fa).

There's unfortunately no good way to choose this number so we can only evolve it over time.

Anyway I do have an idea that MTV could use these options, and in fact tie them to the `virt_v2v_container_limits*` settings, but it's not very well formed at the moment.  We do however need the options in virt-v2v if we're ever going to do that.

Reply:
Oh yeah 100% we need a way to configure it, I was thinking of the default behaviour, whether it should be here or in the libguestfs itself. But guess the selinux relabeling is not common for the guestfs use case.